client/sql/dataframereader.go

```diff
@@ -0,0 +1,41 @@
+package sql
+
+import proto "github.com/apache/spark-connect-go/v_3_4/internal/generated"
+
+type DataFrameReader interface {
```

Comment:
Same for doc here and below for the methods.

Reply:
Yes!